exchange/bitswap/workers.go

```diff
@@ -89,7 +89,7 @@ func (bs *Bitswap) provideWorker(px process.Process) {
 		defer cancel()
 
 		if err := bs.network.Provide(ctx, k); err != nil {
-			log.Error(err)
+			//log.Error(err)
```

Comment:
whoops, temp code i forgot to remove. Was running with no bootstrap peers and bitswap telling me that constantly was rather annoying.